1.0/docs/start/first-element/step-2.md

```diff
@@ -0,0 +1,255 @@
+    defining (in this case, the `<icon-toggle>`). This is the element
+    that contains or <em>hosts </em>the local DOM tree.
+
+
+<p class="alert alert-info">
+The first &lt;template> tag inside a &lt;dom-module> defines the local DOM for
+the element. Any sibling &lt;template> tags are ignored by the library.
+</p>
+
+At the end of the element definition is some JavaScript that registers the
+element. If the element has a `<dom-module>`, this script is usually placed <em>inside</em> the `<dom-module>` to keep everything together.
+
+
+#### Starting code—element registration
+{: .caption }
+
+<pre class="prettyprint">
+  &lt;script>
+    Polymer({
+      /* this is the element's prototype */
+      is: 'icon-toggle'
+    });
+  &lt;/script>
+&lt;/dom-module>
+</pre>
+
+
+Key information:
+
+  * The `Polymer` call <em>registers</em> the element so it's recognized by the browser.
+  * The argument to the Polymer call is the new element's prototype. You'll do more
+with this in a later step.
+  * The `is` property on the prototype is the new element's name. It has to <em>match</em> the `id` on the `<dom-module>` that contains the element's template.
+
+### Create the local DOM structure
+
+Now that you're familiar with the basic layout of the element, add something
+useful to its local DOM template.
+
+Find the `<span>` below the  `local DOM goes here` comment:
+
+#### icon-toggle.html—before
+{: .caption }
+
+<pre class="prettyprint">
+    &lt;!-- local DOM goes here -->
+    &lt;span>Not much here yet.&lt;/span>
+  &lt;/template>
+</pre>
+
+ Replace the `<span>` and its contents with the `<iron-icon>` tag below:
+
+#### icon-toggle.html—after
+{: .caption }
+
+<pre class="prettyprint">
+    &lt;!-- local DOM goes here -->
+    &lt;iron-icon icon="polymer">
+    &lt;/iron-icon>
+  &lt;/template>
+</pre>
+
+Key information:
+
+  * The `<iron-icon>` element is a custom element that renders an icon. Here it's hard-coded to use
+an icon named "polymer".
+
+### Style the local DOM
+
+There are a number of new CSS selectors to work with local DOM. The `icon-toggle.html `file already includes a `:host` selector, discussed earlier, to style the top-level `<icon-toggle>` element.
+
+To style the `<iron-icon>` element, add the following CSS inside the `<style>` tag after the existing content:
+
+#### icon-toggle.html
+{: .caption }
+
+<pre class="prettyprint">
+    &lt;style>
+      /* local styles go here */
+      :host {
+        display: inline-block;
+      }
+      iron-icon {
+        fill: rgba(0,0,0,0);
+        stroke: currentcolor;
+      }
+      :host([pressed]) iron-icon {
+        fill: currentcolor;
+      }
+    &lt;/style>
+</pre>
+
+Key information:
+
+*   The `<iron-icon>` tag uses an SVG icon. The `fill`
+    and `stroke` properties are SVG-specific CSS properties. They
+    set the fill color and the outline color for the icon, respectively.
+
+*   The `:host()` function matches the host element <em>if the
```

Comment:
It sort of occurred to me that someone might not realize that :host is a standard and not just something we made up. Maybe the first time we introduce it we should link to the shadow spec?